docs/static/ds6/program-badges.svg

```diff
@@ -29,6 +29,10 @@
         <path fill-rule="evenodd" stroke="#3665F3" d="M25.9055 1.37253c-1.3052-.52982-2.7713-.4945-4.0532.10597l-15.4 7.21-.1933.09672C4.86466 9.528 3.9869 10.9996 4.00015 12.6085l.12015 15.35-.00127.4631c.01669 4.7347.96358 7.9271 3.51527 11.2829 3.448 4.6 8.988 7.88 16.366 9.294 7.378-1.414 12.918-4.694 16.364-9.294l.2421-.3243c2.4507-3.347 3.3165-6.5857 3.2739-11.4217l.1187-15.4045-.0025-.164c-.0687-1.5952-1.0097-3.0262-2.4502-3.7015l-15.398-7.21-.2428-.10597zm.8703 26.35267c-1.4193-.8138-2.9904-1.3224-4.5475-1.8154-.2822-.0896-.5669-.1741-.8517-.2587-1.2394-.368-2.4812-.7367-3.5301-1.5273C16.6388 23.1535 16 21.6746 16 20.1566c0-2.6094 1.6543-4.6384 4.0245-5.5909.7406-.2993 1.5371-.495 2.3815-.6123l.004-1.4633c.0326-1.9862 3.142-1.9875 3.184 0v1.4496c1.6449.2484 2.8287.9194 4.0285 1.8975.8623.7023 1.4872 1.5004 1.8904 2.4512.4172.982-.3513 2.0677-1.4373 2.097-.6667.0176-1.2696-.3873-1.5052-1.0016-.7186-1.8623-2.0182-2.8893-4.5514-2.8893-2.1014 0-4.807.8345-4.807 3.3491 0 2.2105 1.9577 2.7547 3.7275 3.2467.2568.0714.5097.1417.7521.2158.2737.0837.5493.1648.8252.246.7948.2339 1.5927.4687 2.3588.7693 1.1499.4519 2.3556.9722 3.2878 1.7899 2.0154 1.7704 2.52 4.9133.7387 7.0405-1.3537 1.6092-3.343 2.1221-5.3081 2.5274v1.8056c0 2.0209-3.0982 2.0197-3.0982 0v-1.8056s-1.6566-.3942-2.2298-.6416c-1.4765-.6365-2.7754-1.7232-3.5394-3.1496-.5489-1.0231.2496-2.2574 1.4294-2.2574.6308 0 1.1937.3736 1.4433.9429.8023 1.8243 2.8139 2.5215 4.6892 2.5215 1.8222 0 4.4334-.7861 4.4656-3.0086.018-1.0975-1.1059-1.8585-1.9783-2.3612z" clip-rule="evenodd"/>
     </symbol>
     <symbol id="program-badge-money-back-guarantee-zl" viewBox="0 0 48 48">
-        <path d="M21.8526 0.478988C23.2146 -0.159012 24.7846 -0.159012 26.1486 0.478988L41.5466 7.68899C43.0526 8.39499 44.0126 9.92699 44.0006 11.609L43.8806 26.959C43.9246 31.951 43.0006 35.241 40.3646 38.705C36.9186 43.305 31.3786 46.585 24.0006 47.999C16.6226 46.585 11.0826 43.305 7.63464 38.705C5.00064 35.241 4.07664 31.951 4.12064 26.959L4.00064 11.609C3.98664 9.92699 4.94664 8.39499 6.45264 7.68899L21.8526 0.478988ZM31 11C30.4477 11 30 11.4477 30 12V21.4942L28.5935 22.1204C28.089 22.345 27.8621 22.9361 28.0867 23.4407C28.3113 23.9452 28.9024 24.1721 29.407 23.9475L30 23.6835V33C30 33.5523 30.4477 34 31 34C31.5523 34 32 33.5523 32 33V22.793L33.9747 21.9138C34.4793 21.6892 34.7062 21.098 34.4815 20.5935C34.2569 20.089 33.6658 19.8621 33.1612 20.0867L32 20.6037V12C32 11.4477 31.5523 11 31 11ZM15 18C14.4477 18 14 18.4477 14 19C14 19.5523 14.4477 20 15 20H23.7552L14.2597 32.3277C14.0984 32.5053 14 32.7411 14 33C14 33.5523 14.4477 34 15 34H26C26.5523 34 27 33.5523 27 33C27 32.4477 26.5523 32 26 32H16.9212L26.6955 19.7216L26.705 19.7092C26.8872 19.5281 27 19.2772 27 19C27 18.4477 26.5523 18 26 18H15Z" >
```

Comment:
Was this the whole issue? It was just missing a closing `/`? If so, it would be really helpful if there was some sort of linter to catch these sorts of things.

Reply:
Yes, that was the issue. Since it wasn't closed you couldn't add anything after it.
Would be good to add a linter to it